client/components/open/integrations/GoogleSheetsIntegration.vue

```diff
@@ -5,7 +5,25 @@
     :form="form"
   >
     <div class="my-5">
-      Coming Soon...
+      <select-input
+        v-if="providers.length"
+        v-model="integrationData.oauth_id"
+        name="provider"
+        :options="providers"
+        option-key="id"
+        emit-key="id"
+        :required="true"
+        label="Select Google Account"
+      />
+
+      <v-button
```

Comment:
Here's where I mention that after connecting an account for the first time the "flow" is broken